src/game.h

```diff
@@ -383,7 +384,7 @@ class game
         void nuke( const tripoint &p );
         bool spread_fungus( const tripoint &p );
         std::vector<faction *> factions_at( const tripoint &p );
-        int &scent( const tripoint &p );
+        scent_map scent; // The scent map
```

Comment:
Making it `unique_ptr<scent_map>` would allow getting rid of scent_map.h inclusion. game.h is included everywhere and most of the files it is included in won't need to ever access any of the scent map's methods or fields.